src/scenes/lobbyscreen.ts

```diff
@@ -0,0 +1,99 @@
+import { Nullable, UUID, int } from "definitions/utils";
+import Phaser from "phaser";
+import { log } from "utilities/logger";
+import SessionManager from "managers/SessionManager";
+import PlayerManager from "managers/PlayerManager";
+import { SessionInformation } from "utilities/InformationTypes";
+import { interactify } from "utilities/utils";
+
+export class LobbyScreen extends Phaser.Scene {
+    private text: Nullable<Phaser.GameObjects.Text>;
+    private players: string[];
+    private me: SessionInformation;
+    private graphics!: Phaser.GameObjects.Graphics;
+
+    public constructor() {
+        super("LobbyScreen");
+        this.text = null;
+        // array of players
+        this.players = PlayerManager.getPlayers();
+        // sessionId, playername, playerId, role, turnIndex
+        this.me = SessionManager.getSessionInformation();
+    }
+
+    public syncPlayers = () => {
+        log("In syncPlayers");
+        this.players = PlayerManager.getPlayers();
+        this.update();
+    }
+
+    public syncMe = () => {
+        log("In syncMe")
+        this.me = SessionManager.getSessionInformation();
+        this.update()
+    }
+
+    public init(): void {
+        const playerManagerListener: UUID = PlayerManager.onSync.on(this.syncPlayers);
+        const sessionManagerListener: UUID = SessionManager.onSync.on(this.syncMe);
+
+        // on scene destroy free listener
+        this.events.on("destroy", () => {
+            PlayerManager.onSync.off(playerManagerListener);
+            SessionManager.onSync.off(sessionManagerListener);
+        });
+    }
+    
+    public preload(): void{
+        this.load.image("testButton", "assets/testButton.png");
+    }
+
+    public create(): void {
+        this.text = this.add
+            .text(350, 80, "Saboteur Lobby", {
+                fontFamily: "Goudy",
+                fontSize: 38,
+                color: "#ffffff",
+                stroke: "#000000",
+                strokeThickness: 8,
+                align: "center",
+            })
+            .setOrigin(0.5);
+        log(this, this.text);
+
+        const startButton: Phaser.GameObjects.Image = this.add.image(150, 540, "testButton");
+        startButton.setInteractive();
+        interactify(this, startButton, 1, () => { log("Click brown button"); });
+    }
+
+    private playerBackground(ypos: integer): void{
+        // Placeholder for the background on player display
+        this.graphics = this.add.graphics();
+        this.graphics.lineStyle(1, 0x000000, 1);
+        this.graphics.fillStyle(0x5C4033, 1.0);
+        // xpos, ypos, horizontal length, vertical length, corner radius
+        this.graphics.fillRoundedRect(100, ypos, 600, 27, 5);
+        this.graphics.strokeRoundedRect(100, ypos, 600, 27, 5);
+    }
+
+    public override update(): void {
```

Comment:
attention the method called update is phaser reserved and will be called every frame... you need to rename it to something else.